[2024-03-09T15:43:45.955+0530] {processor.py:161} INFO - Started process (PID=4631) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:43:45.956+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:43:45.956+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:43:45.956+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:43:45.960+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:43:45.959+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:43:45.960+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:43:45.960+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:43:45.960+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:43:45.970+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-09T15:44:49.687+0530] {processor.py:161} INFO - Started process (PID=4689) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:44:49.689+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:44:49.690+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:44:49.689+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:44:49.694+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:44:49.693+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:44:49.694+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:44:49.694+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:44:49.696+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:44:49.730+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2024-03-09T15:45:44.929+0530] {processor.py:161} INFO - Started process (PID=4749) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:45:44.931+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:45:44.932+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:45:44.932+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:45:44.936+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:45:44.936+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:45:44.937+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:45:44.937+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:45:44.937+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:45:44.947+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-03-09T15:46:34.295+0530] {processor.py:161} INFO - Started process (PID=4793) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:46:34.297+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:46:34.298+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:46:34.297+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:46:34.301+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:46:34.300+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:46:34.301+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:46:34.301+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:46:34.301+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:46:34.312+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-03-09T15:47:15.774+0530] {processor.py:161} INFO - Started process (PID=4834) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:47:15.777+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:47:15.778+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:47:15.777+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:47:15.781+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:47:15.780+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:47:15.781+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:47:15.781+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:47:15.782+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:47:15.792+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-03-09T15:48:21.454+0530] {processor.py:161} INFO - Started process (PID=4912) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:48:21.457+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:48:21.458+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:48:21.458+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:48:21.462+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:48:21.461+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:48:21.462+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:48:21.462+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:48:21.462+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:48:21.473+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-03-09T15:49:10.714+0530] {processor.py:161} INFO - Started process (PID=4976) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:49:10.717+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:49:10.718+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:10.718+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:49:10.722+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:10.721+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:49:10.723+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:10.722+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:49:10.723+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:49:10.735+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-03-09T15:49:46.856+0530] {processor.py:161} INFO - Started process (PID=5015) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:49:46.859+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:49:46.861+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:46.860+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:49:46.864+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:46.864+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:49:46.864+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:46.864+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:49:46.865+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:49:46.876+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-03-09T15:50:47.088+0530] {processor.py:161} INFO - Started process (PID=5080) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:50:47.091+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:50:47.093+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:47.092+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:50:47.097+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:47.097+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:50:47.098+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:47.098+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:50:47.098+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:50:47.110+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-03-09T15:51:35.112+0530] {processor.py:161} INFO - Started process (PID=5132) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:51:35.115+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:51:35.116+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:51:35.116+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:51:35.120+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:51:35.120+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:51:35.121+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:51:35.121+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:51:35.121+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:51:35.131+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-03-09T15:52:12.318+0530] {processor.py:161} INFO - Started process (PID=5171) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:52:12.321+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:52:12.322+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:52:12.322+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:52:12.325+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:52:12.325+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:52:12.325+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:52:12.325+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:52:12.326+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:52:12.336+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-03-09T15:53:03.037+0530] {processor.py:161} INFO - Started process (PID=5223) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:53:03.041+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:53:03.043+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:03.042+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:53:03.048+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:03.048+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:53:03.049+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:03.049+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:53:03.049+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:53:03.063+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2024-03-09T15:53:39.310+0530] {processor.py:161} INFO - Started process (PID=5260) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:53:39.312+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:53:39.313+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:39.312+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:53:39.317+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:39.317+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:53:39.317+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:39.317+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:53:39.318+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:53:39.329+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-03-09T15:54:15.631+0530] {processor.py:161} INFO - Started process (PID=5298) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:54:15.634+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-09T15:54:15.635+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:54:15.635+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:54:15.638+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:54:15.638+0530] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-09T15:54:15.639+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:54:15.638+0530] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-09T15:54:15.639+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-09T15:54:15.649+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
